skycoin/skywire
# fix various issues

Did you run `make format && make check`? Yes

Fixes #1779 #1769 #1753 #1694

 Changes:	
- Fix issues on Windows and Mac installer based on v1.3.20-rc1
- Fix start and stop issue on proxy/vpn (include ctrl+c command)
- Fix `--timeout` flag on start proxy/vpn
- Fix `--http` flag to skysocks to serve httpproxy beside socks5

How to test this PR:
You can run visor and trying to start/stop proxy and vpn by these commands:
- `skywire cli proxy start --timeout 30 --http :1010`
- `skywire cli vpn start --timeout 20`
- check `ctrl+c` command during running apps
- For fixing installers, I'll push `v1.3.20-rc2` after merging this PR.
